pkgs/development/interpreters/lua-5/build-luarocks-package.nix

```
@@ -111,6 +112,12 @@ let
   # explicitly inherit this for it to be available as a shell variable in the
   # builder
   rocksSubdir = "${self.pname}-${self.version}-rocks";
+
+  configFile = writeTextFile {
+    name = pname + "-luarocks-config";
```

suggestion(probably doesn't matter):

```suggestion
    name = pname + "-luarocks-config.lua";
```

---

good suggestion ty